### DISCUSSION TITLE
Disponibilização das atas de reuniões

### DISCUSSION THREAD

**Discussion by rennerocha** [apyb/comunidade#22] [Geral]
Olá! Eu achei muito legal as atas das reuniões estarem disponíveis de alguma maneira. Agora, como associado, estou conseguindo acompanhar o que a diretoria está fazendo (junto com as discussões) o que é muito bom! :tada: 

Eu só não achei que o local onde essas atas estão armazenadas (como issues em https://github.com/apyb/tarefas/issues?q=label%3AReuni%C3%A3o+) tão simples de as pessoas descobrirem que elas existem.

Algumas ideias de como melhorar:
- Criar um repositório apyb/reunioes e criar um arquivo MD de cada reunião (com o nome tipo `YYYY-MM-DD_ReuniaoDiretoria.md` ou algo parecido)
- Colocar esses arquivos no apyb/tarefas seguindo o padrão de nomes

Uma vantagem é que se houver alguma mudança no conteúdo da ata de algum dia, um PR pode ser feito e manter o histórico disso de maneira mais transparente.

**Comment by anadulce** [comment]
Boa noite, @rennerocha! Tudo bem?

Ficamos muito felizes por saber que a ideia de trazer as atas abertas das reuniões tem colaborado para melhor visualização das ações da diretoria, e super gostaríamos que fosse ainda mais acessível!
Achamos a sua sugestão muito boa, principalmente no ponto da padronização.
O @rougeth criou templates de issues para o GitHub da Org da Python Brasil (https://github.com/pythonbrasil/pybr2022-org).
Você acha que isso poderia ajudar a tornar as atas mais visíveis?
Ou realmente é uma questão de que issues fechadas ficam num local muito específico?

**Reply by rougeth** [reply]
Eu vou acabar criando um template de issues para reunião para coletar os tópicos a serem discutidos durante a semana. É comum eu lembrar de algum tópico que queria discutir na reunião mas acabar esquecendo por não ter anotado em nenhum lugar.
